contracts/core/EntryPoint.sol

```diff
@@ -27,7 +27,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
 
     using UserOperationLib for UserOperation;
 
-    SenderCreator private senderCreator = new SenderCreator();
+    SenderCreator private immutable senderCreator = new SenderCreator();
```

Comment:
unrelated to memory safety, but nonetheless small optimization for contract creation

Reply:
removed SenderCreator from this commit...